assignments/ex2/ERRATA.md

```diff
@@ -1 +1,3 @@
 - Warn that `lms-verify` should be cloned outside the exercise project.
+- Correct missing test case on interpreter
+- Warn that virtualized `if` statements don't work in lambdas
```

Comment:
Let's take the warning offline by email. Can you just have the correct missing test case, and I'll merge. If you can rebase, that's even better.

Reply:
I've changed it to just the test case fix.